Simplenote/NoteEditorViewController.m

```diff
@@ -1091,7 +1079,9 @@ - (void)applyStyle
 
 - (void)showPublishPopover
 {
-    [self showViewController:self.publishViewController relativeToView:shareButton preferredEdge:NSMaxYEdge];
+    PublishViewController *viewController = [PublishViewController new];
+    [self showViewController:viewController relativeToView:shareButton preferredEdge:NSMaxYEdge];
+    self.publishViewController = viewController;
```

Comment:
Betcha you're gonna wanna do something like `self.publishViewController.delegate = self` right here yo. ;)